### ISSUE TITLE
unexcepted  result return with the inference code

### ISSUE THREAD

**Issue by cvtower** [NVlabs/Sana#72]
i just setup sana follow the script file environment_setup.sh , run the provided inference code:

import torch
from app.sana_pipeline import SanaPipeline
from torchvision.utils import save_image

device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
generator = torch.Generator(device=device).manual_seed(42)

sana = SanaPipeline("configs/sana_config/1024ms/Sana_1600M_img1024.yaml")
sana.from_pretrained("hf://Efficient-Large-Model/Sana_1600M_1024px/checkpoints/Sana_1600M_1024px.pth")
prompt = 'a cyberpunk cat with a neon sign that says "Sana"'

image = sana(
    prompt=prompt,
    height=1024,
    width=1024,
    guidance_scale=5.0,
    pag_guidance_scale=2.0,
    num_inference_steps=18,
    generator=generator,
)
save_image(image, 'output/sana.png', nrow=1, normalize=True, value_range=(-1, 1))

and the logs:
Fetching 10 files: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 10/10 [00:00<00:00, 70969.61it/s]
Loading safetensors
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.

**Comment by Deng-Xian-Sheng** [comment]
You should open the local address in the browser and use it.

I understand that you found that the public address is not returned, which may be some kind of problem, but you can use the local address as a compromise

**Comment by Deng-Xian-Sheng** [comment]
You're in China? That's awesome.

**Comment by lawrence-cj** [comment]
I don't get what's the 'unexcepted result' here? @cvtower

**Comment by cvtower** [comment]
> You should open the local address in the browser and use it.
> 
> I understand that you found that the public address is not returned, which may be some kind of problem, but you can use the local address as a compromise

already checked the address several times, and the result is:
the webpage shows another repo-OmniGen, another file folder as well as python lib in my conda environment.

and i just grep key words this sana source code, no 'OmniGen' related file found.
maybe it is caused by the gradio or other requirement libs by sana?

any tips will be appreciated since we cant wait to compare sana with other works.

**Comment by cvtower** [comment]
> I don't get what's the 'unexcepted result' here? @cvtower

here, with the inference code from readme file:

####
import torch
from app.sana_pipeline import SanaPipeline
from torchvision.utils import save_image

device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
generator = torch.Generator(device=device).manual_seed(42)

sana = SanaPipeline("configs/sana_config/1024ms/Sana_1600M_img1024.yaml")
sana.from_pretrained("hf://Efficient-Large-Model/Sana_1600M_1024px/checkpoints/Sana_1600M_1024px.pth")
prompt = 'a cyberpunk cat with a neon sign that says "Sana"'

image = sana(
    prompt=prompt,
    height=1024,
    width=1024,
    guidance_scale=5.0,
    pag_guidance_scale=2.0,
    num_inference_steps=18,
    generator=generator,
)
save_image(image, 'output/sana.png', nrow=1, normalize=True, value_range=(-1, 1))
#####

but seems like the sana model not loaded, and log is :

####
Fetching 10 files: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 10/10 [00:00<00:00, 70969.61it/s]
Loading safetensors
Running on local URL: http://127.0.0.1:7860/

To create a public link, set share=True in launch().
####

and the local web shows another repo-OmniGen. if needed, i will upload screenshot or video.

**Comment by lawrence-cj** [comment]
Can you upload a screenshot or a video for me to better undetstand?

**Comment by cvtower** [comment]
Since diffusers support released.
I will close this issue and no longer use the repo inference source.

Thanks for all ur help.
